### ISSUE TITLE
pkg-config pc files not created for *_threads libs

### ISSUE THREAD

**Issue by cimes-isi** [FFTW/fftw3#180]
It appears that pc files are not being created (or at least installed) for the libraries that provide thread-related functions, which makes it difficult to automatically discover if these libraries are available.  PR #52 describes nicely FFTW's position on threading, but since functions like `fftw[f]_threads_init` are provided by the FFTW *_threads libraries, those FFTW libraries need to be discoverable, even if their pkg-config files leave the actual thread linking to the application (e.g., `-lpthread`).

**Comment by matteo-frigo** [comment]
I think that the right approach to this problem is to say that installing libFOO_threads.{a.so} installs a separate file FOO_threads.pc (as opposed to adding the threads library to the normal fftw3.pc file).
Does it sound reasonable?

**Comment by cimes-isi** [comment]
Yes, that is what I'd expect.  `FOO_threads.pc` should in turn have a `Requires` (or perhaps `Requires.private`) on `FOO` since `libFOO.{a,so}` is a transitive dependency.

**Comment by samueljmcameron** [comment]
I would be interested in this feature as well, though mainly for mpi, not threading.

**Comment by antoniovazquezblanco** [comment]
I am currently dealing with this issue. It seems to get more complicated when fftw is installed via brew in a non-standard location and the compiler does not find the library by itself so meson build system needs extra configuration...

It would be awesome to see this fixed...

**Comment by antoniovazquezblanco** [comment]
A possible fix at https://github.com/FFTW/fftw3/pull/407
